.github/workflows/rust.yml

```diff
@@ -2,9 +2,9 @@ name: Rust
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
```

Comment:
⛏️ The brackets are unnecessary but I would surround with double quotes.

Reply:
Cleaned up in [4588d8f](https://github.com/bitwarden/bitwarden-russh/pull/5/commits/4588d8f4afd966fbd7be259cf3c24dd4999c3a97)